### DISCUSSION TITLE
Repository for beta install within Unraid

### DISCUSSION THREAD

**Discussion by DrSpaldo** [blakeblackshear/frigate#1754] [Ask A Question]
When creating the container/app in Unraid you need to pick the repository name, ie. blakeblackshear/frigate:stable-amd64

If I would like for it to pull the latest beta, instead of the latest stable release, rather than specifically naming the latest beta each time

**Comment by mitchross** [comment]
All you have to do is change the docker line to blakeblackshear/frigate:0.9.0-rc3-amd64
